django/db/models/base.py

```diff
@@ -460,9 +460,13 @@ class ModelState:
     adding = True
     fields_cache = ModelStateCacheDescriptor()
     related_managers_cache = ModelStateCacheDescriptor()
+    peers = ()
```

Comment:
when changing to a `WeakValueDictionary`, this could be made into an empty immutable mapping with [`MappingProxyType`](https://adamj.eu/tech/2022/01/05/how-to-make-immutable-dict-in-python/) (maybe) or just a normal dict